README.md

```diff
@@ -146,6 +146,12 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
   - Custom load balancing algorithms: You can use custom load balancing algorithms during the `balancer` phase.
   - Custom routing: Support users to implement routing algorithms themselves.
 
+- **Multi-Language support**
+  - Apache APISIX is a multi-language gateway for plugin development and provides support via `WASM` and `RPC`.
+  ![Multi Language Support into Apache APISIX](docs/assets/images/apisix-multi-lang-support.png)
+  - The WASM or WebAssembly, is the modern way. If we look at the left part, APISIX can load and run WASM bytecode via APISIX wasm plugin, so developers only need to write the code according to the SDK and then compile it into a WASM bytecode file and put it in the APISIX folder. As of now, APISIX has support for Golang SDK and Rust SDK - more language SDKs will be supported in the future.
```

Comment:
we can add a link for `Golang SDK` and `Rust SDK` later.

Reply:
Hi @membphis, one question - if the wasm plugin of apache apisix just runs the wasm bytecode why are we limited only to the Go and Rust? At the end of the day, it's the bytecode instructions that we need to run on wasm, right?

https://webassembly.org/getting-started/developers-guide/

This official URL shows the list of languages where the programs can e compiled for wasm virtual machine. Thanks